source/dev-setup.rst

```diff
@@ -59,6 +59,12 @@ extensions::
     brew install postgres
     brew install postgis
 
+Note that "brew install postgis" will install the latest version of postgis along with the latest
+version of potstgres, so if you have a previous installation of postgres you may end up with two
+versions installed causing conflicts and connection issues. It is recommended, if you have a
+previous installation, that you uninstall this version version. Migration of existing data is
+left as an exercise for the reader.
```

Comment:
I don't feel comfortable just hinting that this upgrade could lose the developer's current data without giving them any help in migrating it. That link was just a guess of something that someone who uses Macs regularly might try to see if it would be helpful based on a quick Google search, but I'm obviously not one of those people so have no way to judge myself.